lib/src/executables/sort_intl.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io';
+import 'package:args/args.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:logging/logging.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_messages.dart';
+
+import 'package:over_react_codemod/src/util/package_util.dart';
+import 'package:path/path.dart' as p;
+import '../util.dart';
+
+final _log = Logger('orcm.intl_message_migration');
+
+final FileSystem fs = const LocalFileSystem();
+
+final parser = ArgParser()
+  ..addFlag(
+    'help',
+    abbr: 'h',
+    negatable: false,
+    help: 'Prints this help output.',
+  );
+
+late ArgResults parsedArgs;
+
+void main(List<String> args) async {
+  parsedArgs = parser.parse(args);
+  if (parsedArgs['help'] as bool) {
+    printUsage();
+    return;
+  }
+
+  // If we have specified paths on the command line, limit our processing to
+  // those, and make sure they're absolute.
```

Comment:
So there are two directions this could go.
1) Don't find all the Dart files. Find all the _intl.dart files under the root directory, but exclude test. Then I think this code would more or less work.
2) Don't do any of this.  Do what I said the last time, which you marked as resolved without doing it or explaining. The only difficulty is that it requires the package name. But the only thing that's used for is to name the intl file. So you could derive the package name from the pubspec as before, but knowing that it's in the root directory and there's only one. Or you could derive it from the name of the intl.dart file, whose position is known.

Reply:
@alanknight-wk I have updated the code 
1. To derive the package name from `pubspec.yaml ` create the following [method](https://github.com/Workiva/over_react_codemod/blob/413a24d3bf43f06a44ea1227288bd515dcf99326/lib/src/executables/sort_intl.dart#L56) 
2. Also tested in **ml_ui_client** if we run this executable at root level where no `pubspec.yaml` this shows an error 
**could not determine the package name** but under `ml_ui_client`  directory it works .
3. Also tested in DPC where we have `pubspec.yaml` on root it not shows error but sorting not working because **intl.dart** files are under `subpackage/package/lib/src`  eg: `subpackage/datatables/lib/src`